instana/collector/helpers/runtime.py

```diff
@@ -181,7 +182,7 @@ def _collect_runtime_snapshot(self, plugin_data):
 
             if 'AUTOWRAPT_BOOTSTRAP' in os.environ:
                 snapshot_payload['m'] = 'Autowrapt'
-            elif 'INSTANA_MAGIC' in os.environ:
```

Comment:
Is the `INSTANA_MAGIC` variable in the `instana/instrumentation/fastapi_inst.py` still needed?

Reply:
That is a different can of worms that belongs to a separate PR.